docs/docs/guides/become-an-operator.md

```diff
@@ -0,0 +1,141 @@
+
+The only information they will need from you is your address. Please do not engage with any accounts claiming to be Support, Admin or Help. Report any account that is asking you to sign transactions or asking for any sort of credentials such as your private key. These accounts are trying to steal your tokens.
+
+### Mumbai testing environment node config:
+```json
+{
+    "client": {
+        "metrics": false,
+        "auth": {
+            "privateKey": "NODE_PRIVATE_KEY"
+        },
+        "network": {
+            "controlLayer": {
+                "entryPoints": [
+                    {
+                        "id": "e1",
+                        "websocket": {
+                            "host": "entrypoint-1.streamr.network",
+                            "port": 40401,
+                            "tls": true
+                        }
+                    },
+                    {
+                        "id": "e2",
+                        "websocket": {
+                            "host": "entrypoint-2.streamr.network",
+                            "port": 40401,
+                            "tls": true
+                        }
+                    }
+                ]
+            }
+        },
+        "contracts": {
+            "streamRegistryChainAddress": "0x4F0779292bd0aB33B9EBC1DBE8e0868f3940E3F2",
+            "streamStorageRegistryChainAddress": "0xA5a2298c9b48C08DaBF5D76727620d898FD2BEc1",
+            "storageNodeRegistryChainAddress": "0xE6D449A7Ef200C0e50418c56F84079B9fe625199",
+            "mainChainRPCs": {
+                "name": "mumbai",
+                "chainId": 80001,
+                "rpcs": [
+                    {
+                        "url": "https://rpc-mumbai.maticvigil.com"
+                    }
+                ]
+            },
+            "streamRegistryChainRPCs": {
+                "name": "mumbai",
+                "chainId": 80001,
+                "rpcs": [
+                    {
+                        "url": "https://rpc-mumbai.maticvigil.com"
+                    }
+                ]
+            },
+            "theGraphUrl": "https://api.thegraph.com/subgraphs/name/samt1803/network-subgraphs"
+        }
+    },
+    "plugins": {
+        "operator": {
+            "operatorContractAddress": "YOUR_OPERATOR_CONTRACT_ADDRESS"
+        }
+    }
+}
+```
+
+## Testnet configuration
+Editing the `config/default.json` is the main way to configure your node.
+
+To particpate in the incentivized testnets, you'll need to add a few extra lines of config to your node. That config can be found on the [dedicated testnet page](../streamr-testnets/testnets.md) closer to the date of the testnets.
```

Comment:
Typo + comment: maybe any other configs should be on this page, since the Mumbai config is on this page already?

```suggestion
To participate in the incentivized testnets, you'll need to add a few extra lines of config to your node. That config can be found on the [dedicated testnet page](../streamr-testnets/testnets.md) closer to the date of the testnets.
```